adapters/eplanning/eplanning.go

```diff
@@ -61,12 +66,47 @@ type hbResponseAd struct {
 	Height       uint64 `json:"h,omitempty"`
 }
 
+type byPriority []openrtb.Format
```

Comment:
I recommend moving the sorter to its own file due to its non-trivial logic. I'd also recommend coded unit tests (not using the json framework) for this sorting behavior specifically.